spec/Appendix B -- Grammar Summary.md

```diff
@@ -224,7 +224,9 @@ ObjectTypeExtension :
   - extend type Name ImplementsInterfaces? Directives[Const]
   - extend type Name ImplementsInterfaces
 
-ImplementsInterfaces : implements NamedType+
+ImplementsInterfaces :
+  - implements `&`? NamedType
```

Comment:
Where did the ampersand syntax come from? Has this been implemented anywhere and found to be preferable over simple whitespace/comma separation? 🤔 

Looks like graphql-js just uses whitespace/commas:
https://github.com/graphql/graphql-js/blob/master@{2017-12-19}/src/language/parser.js#L921-L923

Reply:
This solves an ambiguity we should have caught sooner but was found in practice.

https://github.com/graphql/graphql-js/pull/1166 documents the ambiguity

https://github.com/graphql/graphql-js/pull/1169 implements this same proposed fix